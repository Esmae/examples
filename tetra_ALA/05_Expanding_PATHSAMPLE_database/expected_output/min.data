      -35.350347228396458     9790.913915977564102     1      293.6823691158      860.7846898767     1024.4566946858
      -41.058341717437770     9792.881833598055891     1      324.7291569216      655.0519707341      671.7053052785
      -41.722992401385000     9796.924653787169518     1      399.2855477052      500.2110330140      532.1986393602
      -34.565803637572685     9793.123053104787687     1      342.7178657947      684.4834636497      914.7404945813
      -34.643081859423830     9792.308212005182213     1      417.7447257063      506.5081170220      808.2370917033
      -37.972810010183423     9793.625213920353417     1      352.9233578682      700.8860555488      763.1411834009
      -34.602207579854792     9790.004039502335218     1      282.0982909618     1007.9568813746     1161.5432441878
      -37.978229658195403     9793.267796121168431     1      347.8674092629      646.3319483661      731.0186132524
      -37.173833653352972     9791.141860436682691     1      444.0061201782      568.7824715754      789.8291026186
      -35.309321206578602     9795.049143313635795     1      403.8632073857      517.1911622086      779.9206934305
      -37.121234164494354     9787.391643072474835     1      315.6042141232      740.5547627312      867.1925811728
      -36.155529379948632     9796.999761064171253     1      300.8420532578      697.8792714324      826.9275860347
      -35.184894689712223     9793.976973761344198     1      332.7238330512      672.5852522950      794.0561444218
      -37.072147322882302     9791.617778112082306     1      458.5262941441      566.4838062509      816.7647242055
      -36.474285209159291     9798.555612983529500     1      329.5658775860      564.3151053381      698.2965783249
      -35.909405093084388     9793.712235802453506     1      332.0502780870      756.8151517037      907.7852458368
      -31.986356017495893     9794.915467914433975     1      404.2736531193      468.9945665539      701.3525224860
      -34.641031850578088     9792.240771575701729     1      356.0890425030      588.4991843850      780.0381590321
      -32.870065152182065     9795.482039277943841     1      371.2166747433      512.2180140534      669.2798044122
      -34.863923711860764     9790.045097247262675     1      270.2679104005     1021.8081563606     1058.1579528840
      -32.576171257003466     9791.291401683078220     1      354.4624799669      628.5197806667      770.9931865571
      -37.150749020109970     9794.563390376750249     1      309.7627704484      628.0636614472      779.9600191653
      -39.443905062615613     9798.126886132471554     1      422.1187397875      446.4718859913      541.7943367177
