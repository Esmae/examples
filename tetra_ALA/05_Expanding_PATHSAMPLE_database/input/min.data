      -35.350347228396458     9790.913915977564102     1      293.6823691158      860.7846898767     1024.4566946858
      -41.058341717437770     9792.881833598055891     1      324.7291569216      655.0519707341      671.7053052785
      -41.722992401385000     9796.924653787169518     1      399.2855477052      500.2110330140      532.1986393602
      -34.565803637572685     9793.123053104787687     1      342.7178657947      684.4834636497      914.7404945813
      -34.643081859423830     9792.308212005182213     1      417.7447257063      506.5081170220      808.2370917033
      -37.972810010183423     9793.625213920353417     1      352.9233578682      700.8860555488      763.1411834009
      -34.602207579854792     9790.004039502335218     1      282.0982909618     1007.9568813746     1161.5432441878
      -37.978229658195403     9793.267796121168431     1      347.8674092629      646.3319483661      731.0186132524
      -37.173833653352972     9791.141860436682691     1      444.0061201782      568.7824715754      789.8291026186
      -35.309321206578602     9795.049143313635795     1      403.8632073857      517.1911622086      779.9206934305
      -37.121234164494354     9787.391643072474835     1      315.6042141232      740.5547627312      867.1925811728
      -36.155529379948632     9796.999761064171253     1      300.8420532578      697.8792714324      826.9275860347
