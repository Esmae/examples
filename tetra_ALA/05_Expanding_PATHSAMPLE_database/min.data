      -34.643081859411154     9792.308238212928700     1      417.7447075320      506.5081099182      808.2370354078
      -34.641031850564644     9792.240808557911805     1      356.0890598633      588.4991576352      780.0381531537
      -35.746615918092346     9787.407236337543509     1      264.2367431521     1031.2925542179     1166.4390465570
      -37.972810010173589     9793.625239405811953     1      352.9233608440      700.8861365039      763.1412147773
      -40.644884312187685     9790.932799240330496     1      351.9761969266      645.5059666473      727.9672182984
      -41.722992401373233     9796.924678656811011     1      399.2855525698      500.2110046061      532.1986258872
      -37.978229658185398     9793.267826482619057     1      347.8673910792      646.3319507571      731.0186568110
      -34.565803637565637     9793.123083700216739     1      342.7178982912      684.4833548175      914.7404198575
      -34.863923711850816     9790.045119215295927     1      270.2679424328     1021.8080889867     1058.1578448615
      -34.602207579841263     9790.004065099512445     1      282.0982791768     1007.9568323982     1161.5431908777
      -41.233876808604919     9796.883888743275747     1      383.7227911368      503.6297412606      582.3626169080
      -39.480337411474579     9788.853493358405103     1      313.5895520879      932.0189943214      992.1033151057
      -37.031276712099555     9787.433791347968508     1      257.8513833082     1018.3405051178     1127.0194817463
      -35.928144127380989     9790.385554149566815     1      310.5349014236      865.3700527114     1059.8106348448
      -38.682289361648145     9786.815838725024150     1      425.9025825396      783.2237617633     1037.5013038278
      -35.994480232773640     9787.870929745766261     1      362.1585872014     1047.7878409845     1219.3975675380
      -36.603929473335313     9795.096633043052861     1      402.5224314659      504.5918430946      734.0803734599
      -36.709183869100912     9791.246639208060515     1      316.8871733784      724.7490210587      928.4230266174
      -36.044881045490342     9787.419688040381516     1      259.3858655874     1188.6817109420     1259.3833526327
      -38.466723188666890     9791.512760547282596     1      436.8421465315      544.6680252480      713.8180923917
      -35.309321206566551     9795.049164680311151     1      403.8632106207      517.1912188010      779.9207814005
      -37.173833653340409     9791.141887154186406     1      444.0060980957      568.7824740915      789.8290649687
      -38.203378646171302     9784.938205578020643     1      347.3496175490      862.1824375952     1033.8509514241
      -37.073490744509115     9787.747476492115311     1      248.6484528448      982.1181271090     1103.5690603625
      -38.163405315040677     9788.614864681994732     1      288.2812816255      977.0218953867     1111.8077169778
      -38.658795145970601     9785.184887440474995     1      225.0610157410     1346.9942011753     1368.5225189746
      -38.774728829103253     9786.499815371877048     1      398.1698378704      814.6165063439     1034.9715903920
      -39.704578453701856     9788.494770309911473     1      433.1969166773      582.7272277716      787.7687230128
      -38.472563481274847     9785.229251242104510     1      275.1059611002     1168.9715944137     1247.1692299989
      -37.697257122592362     9787.810668813606753     1      250.5902737247     1153.5529837971     1187.5130958222
      -37.812113820655043     9788.245242053479160     1      276.2178014613      871.7370183630      995.3351076942
      -35.350347228385260     9790.913941404580328     1      293.6823794107      860.7844965376     1024.4564727246
      -35.913910545216595     9787.227914699626126     1      363.1343035204      996.4283839928     1160.2324373063
      -36.005230147499503     9794.060627243359704     1      300.6697797594      699.9415645565      866.6224154377
      -36.186599223161238     9790.426886865616325     1      283.6997114570      894.4341239976      978.8062081040
      -37.442362997851717     9787.871160609403887     1      288.0747336582     1108.4104686657     1231.1351472893
      -38.585007613130529     9785.527972371817668     1      223.7474167393     1377.7429885388     1398.0518740967
      -35.903804471282399     9787.818368773620023     1      233.7845216544     1392.7856341912     1477.6585287076
      -38.511678873716960     9785.508718958686586     1      218.8954935488     1432.0939734673     1462.7316614560
      -37.288175346525350     9787.541771252424951     1      265.1172944628      981.7161027833     1032.5331882660
      -35.813445105538676     9787.242645716185507     1      244.7454842127     1322.9806882083     1416.8827455746
      -38.375293083726604     9785.709320933625349     1      182.8460754005     1559.8499374365     1596.2825562506
      -38.324353466810578     9785.386988424308583     1      207.7789261774     1375.4403185311     1419.8233433668
      -37.264856736701461     9787.774448709307762     1      212.3460222839     1153.4478645562     1216.1853619116
      -39.609268291684657     9789.176331698266949     1      432.2967434038      604.5049927797      820.7058111695
      -38.381856920525479     9792.005162888890482     1      447.2805314177      542.9167981351      739.2159878076
      -38.415409510697373     9785.112028946088685     1      276.8274300501     1089.9259666807     1168.8576418609
      -38.334207493945371     9785.473661394094961     1      278.6922526798     1113.7493090264     1197.7426785642
