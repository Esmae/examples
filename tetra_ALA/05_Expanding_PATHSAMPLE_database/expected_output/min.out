
          -------------------------------------------------------
          Amber 12 SANDER                              2012
          -------------------------------------------------------

| PMEMD implementation of SANDER, Release 12

| Run on 09/18/2019 at 17:45:36

  [-O]verwriting output

File Assignments:
|   MDIN: min.in                                                                
|  MDOUT: min.out                                                               
| INPCRD: coords.inpcrd                                                         
|   PARM: coords.prmtop                                                         
| RESTRT: min.rst                                                               
|   REFC: refc                                                                  
|  MDVEL: mdvel                                                                 
|   MDEN: mden                                                                  
|  MDCRD: mdcrd                                                                 
| MDINFO: mdinfo                                                                


 Here is the input file:

AMBER input file used to return the energy from sander when running GMIN       
 &cntrl                                                                        
  imin   = 1,                                                                  
  ncyc = 1,                                                                    
  maxcyc = 1,                                                                  
  igb = 2,                                                                     
  ntb    = 0,                                                                  
  cut    = 999.99                                                              
 /                                                                             


 
| Conditional Compilation Defines Used:

| New format PARM file being parsed.
| Version =    1.000 Date = 08/04/11 Time = 11:49:36

| Note: 1-4 EEL scale factors were NOT found in the topology file.
|       Using default value of 1.2.

| Note: 1-4 VDW scale factors were NOT found in the topology file.
|       Using default value of 2.0.
| Duplicated    0 dihedrals

| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

 NATOM  =      52 NTYPES =       7 NBONH =      27 MBONA  =      24
 NTHETH =      58 MTHETA =      32 NPHIH =     102 MPHIA  =      87
 NHPARM =       0 NPARM  =       0 NNB   =     258 NRES   =       6
 NBONA  =      24 NTHETA =      32 NPHIA =      87 NUMBND =       8
 NUMANG =      16 NPTRA  =      19 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =      10 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are H(N)-modified Bondi radii (mbondi2)                                             
--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

ACE                                                                             

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =      50, ntrx    =       1, ntwr    =     500
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       0, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       2, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.99000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    0.80000
     gbbeta  =   0.00000, gbgamma =   2.90912, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =       1, ncyc    =       1, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

| Intermolecular bonds treatment:
|     no_intermolecular_bonds =       1

| Energy averages sample interval:
|     ene_avg_sampling =       1

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

ACE                                                                             
 begin time read from input coords =     0.000 ps

 
 Number of triangulated 3-point waters found:        0
| Dynamic Memory, Types Used:
| Reals                1816
| Integers             4647

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

